### ISSUE TITLE
Classical ASP.net project support planned?

### ISSUE THREAD

**Issue by Danielku15** [onovotny/MSBuildSdkExtras#60]
First of all thanks for this project, it really delivers what Microsoft missed when releasing the new csprojs. Since the NuGet SDK support it is also usable without any complex adjustments to the project files anymore. 

The only thing I am a bit missing is the support for classical ASP.net projects. I guess from the file and project settings perspective it should be not a too big deal to add support (some rules for cshtml, aspx, asax etc.). The only thing that maybe is not that easy is telling Visual Studio that this is a web project. As you might know that a web project brings along quite some special GUI settings for debugging and publishing. 

Support for this project type would really fill the gap of maintaining old and new csprojs together in a solution. 

Long story short: Is there a plan to also support ASP.net projects?

**Comment by Nirmal4G** [comment]
see #56

**Comment by Danielku15** [comment]
Sorry but I do not get the point behind the description of the mentioned defect. Is this the current state, something you're currently working on, or the overall roadmap of the project?

**Comment by Nirmal4G** [comment]
As @onovotny mentioned project system doesn't work. I have some workarounds but it requires some extreme hacks. I'm doing full msbuild support for all the classic projects (since I have many of those). I already did WPF, WinForms, ASP.NET is next on my list.

**Comment by Danielku15** [comment]
Thanks for the detailed answer. 😄 

We are already considering to move to ASP.net core in long term but for now it would have been great to have a short/mid-term workaround for our classical project without the risk of changing the underlying technology. I'm closing this issue as it was more a question than a feature request and the question is answered.
